tiny-java-containers/setup-musl.sh

```diff
@@ -10,7 +10,7 @@ rm x86_64-linux-musl-native.tgz
 
 # Download, build, install zlib into TOOLCHAIN_DIR
 echo "zlib version=${ZLIB_VERSION}"
-wget -q https://zlib.net/zlib-${ZLIB_VERSION}.tar.gz
+wget -q https://zlib.net/fossils/zlib-${ZLIB_VERSION}.tar.gz
```

Comment:
BTW: The GitHub Action for GraalVM downloads zlib straight from the GitHub repository:
https://github.com/graalvm/setup-graalvm/blob/4f62eae3cc9579b19bbf2408f993cdd42b6c62fd/src/features.ts#L26-L29